### PR TITLE
added 6_coagulation_brownian

### PR DESCRIPTION
**Pull request by zzheng93**

### CODE REVIEW

File: scenarios/6_coagulation_brownian/postprocessing-demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,215 @@
+{
```
[the remark this answers is not in the document]
Comment (jcurtis2) [reply scenarios/6_coagulation_brownian/postprocessing-demo.ipynb]: I've fixed updated the python code to be two separate scripts: one will plot a size distribution and the other plots the number concentration time series.

**Comment by jcurtis2** [comment]
> Is `scenarios/5_coagulation` actually being used in the current version of the book? If not, I think we should delete that one, and the scenario in this PR should be number 5.

Version in the book is brownian coagulation (this scenario) so I've moved it and deleted the old sedimentation example.

**Review by mwest1066 — APPROVED** [review]

**Comment by mwest1066** [comment]
> Version in the book is brownian coagulation (this scenario) so I've moved it and deleted the old sedimentation example.

Sounds good. Just to confirm, this `5_coag_brownian` is the exact scenario in the book?

Assuming so, go ahead and merge (after fixing the conflict).

**Comment by jcurtis2** [comment]
Conflict was corrected and merged by #164

**Pull request closed without merging**